### PR TITLE
docs: add instructions on how to use design tokens

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé
Added instructions on how to use the GCDS Design Tokens
- https://github.com/cds-snc/design-gc-conception/issues/989

### CODE REVIEW

**Comment by daine** [comment]
PR should be ready to review, we can update the instructions to add the FR translations for the update later

File: README.md [hunk 1]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```js
+import '@cdssnc/gcds-components/gcds.css';
+```
+
+You already have our design tokens available to you! Simply use them in your custom stylesheets like so:
+
+```css
+ font: var(--gcds-font-text);
+ color: var(--gcds-text-primary);
+
```
Comment (melaniebmn) [inline README.md:43]: Sorry very OCD (😆) but can we either remove this line break or add a line break between lines 41 and 42 as well?

File: README.md [hunk 2]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```
+npm i @cdssnc/gcds-tokens
+```
+
+#### Import all GCDS Tokens
+Choose to include all GCDS tokens if you are planning on using the component, global and base tokens. Add all GCDS tokens to your CSS by importing the <tokens.css> file:
+
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/tokens.css'
```
Comment (melaniebmn) [inline README.md:64]: Do we need to  / should we include the package version here?

File: README.md [hunk 3]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```
+
+Add all GCDS tokens in your SCSS by importing the <tokens.scss> file:
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/scss/tokens.scss'
+```
+
+#### Import GCDS component tokens
+Choose to only include GCDS component tokens if you are planning on only using component-specific tokens. Add GCDS component-specific tokens to your CSS by importing the CSS file for a specific component. For example, add the component tokens for the <gcds-button> component by importing the <button.css> file:
```
Comment (melaniebmn) [inline README.md:74]: `<gcds-button>` needs to be changed to ``gcds-button`` to render properly in the readme. We normally only add the <> in documents to signal to translation that this is code and doesn't need to be translated.

Comment (melaniebmn) [inline README.md:74]: `<gcds-button>` and `<button.css>` need to be changed to render properly in the readme. We normally only add the <> in documents to signal to the translation team that this is code and doesn't need to be translated.

```suggestion
Choose to only include GCDS component tokens if you are planning on only using component-specific tokens. Add GCDS component-specific tokens to your CSS by importing the CSS file for a specific component. For example, add the component tokens for the `gcds-button` component by importing the `button.css` file:
```

File: README.md [hunk 4]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+
+#### Install GCDS Tokens (Design Tokens)
+Navigate to the root folder of your project and run:
+
+```
+npm i @cdssnc/gcds-tokens
+```
+
+#### Import all GCDS Tokens
+Choose to include all GCDS tokens if you are planning on using the component, global and base tokens. Add all GCDS tokens to your CSS by importing the <tokens.css> file:
```
Comment (melaniebmn) [inline README.md:60]:
```suggestion
Choose to include all GCDS tokens if you are planning on using the component, global and base tokens. Add all GCDS tokens to your CSS by importing the `tokens.css` file:
```

File: README.md [hunk 5]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/tokens.css'
+```
+
+Add all GCDS tokens in your SCSS by importing the <tokens.scss> file:
```
Comment (melaniebmn) [inline README.md:67]:
```suggestion
Add all GCDS tokens in your SCSS by importing the `tokens.scss` file:
```

File: README.md [hunk 6]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```
+
+#### Import GCDS component tokens
+Choose to only include GCDS component tokens if you are planning on only using component-specific tokens. Add GCDS component-specific tokens to your CSS by importing the CSS file for a specific component. For example, add the component tokens for the <gcds-button> component by importing the <button.css> file:
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/components/button.css'
+```
+
+Add the component tokens in your SCSS for the <gcds-button> component by importing the <button.scss> file:
```
Comment (melaniebmn) [inline README.md:80]:
```suggestion
Add the component tokens in your SCSS for the <gcds-button> component by importing the `button.scss` file:
```

File: README.md [hunk 7]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/scss/components/button.scss'
+```
+
+Repeat the above step for each component-specific CSS or SCSS file you want to include. Or include all component-specific tokens to your CSS by importing the <components.css> file:
```
Comment (melaniebmn) [inline README.md:86]:
```suggestion
Repeat the above step for each component-specific CSS or SCSS file you want to include. Or include all component-specific tokens to your CSS by importing the `components.css` file:
```

File: README.md [hunk 8]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/components.css'
+```
+
+Add all component tokens in your SCSS by importing the <components.scss> file:
```
Comment (melaniebmn) [inline README.md:92]:
```suggestion
Add all component tokens in your SCSS by importing the `components.scss` file:
```

File: README.md [hunk 9]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/scss/components.scss'
+```
+
+#### Import GCDS global and base tokens
+Choose to only include the GCDS global and base tokens if you don’t need component-specific tokens. Add GCDS global and base tokens to your CSS by importing the <base.css> and <global.css> files:
```
Comment (melaniebmn) [inline README.md:99]:
```suggestion
Choose to only include the GCDS global and base tokens if you don’t need component-specific tokens. Add GCDS global and base tokens to your CSS by importing the `base.css` and `global.css` files:
```

File: README.md [hunk 10]
```diff
@@ -10,25 +10,123 @@ Tokens replace hard-coded values with variables that can be reused throughout th
...
+```css
+<!-- GC Design System Tokens -->
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/base.css'
+@import 'path/to/node_modules/@cdssnc/gcds-tokens/build/web/css/global.css'
+```
+
+Add the GCDS global and base tokens to your SCSS by importing the <base.scss> and <global.scss> files:
```
Comment (melaniebmn) [inline README.md:106]:
```suggestion
Add the GCDS global and base tokens to your SCSS by importing the `base.scss` and `global.scss` files:
```

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Added some formatting suggestions :)

File: README.md (hunk 3 above)
Comment (daine) [reply README.md:73]: I didn't know about that and never did it that way, now I do! Thanks for these commit suggestions, makes my job way easier!

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**